queries.sql

```diff
@@ -0,0 +1,9 @@
+
+select * from animals where name like '%mon%';
+select name from animals where date_of_birth between '2016-01-01' and '2019-12-30';
+select name from animals where neutered = true and escape_attempt < 3;
+select date_of_birth from animals where name in ('Agumon', 'Pikachu');
```

Comment:
- [ ] This query returns the wrong result. Kindly check comment ... for the appropriate explanation. 👍 😎